### DISCUSSION TITLE
when do you know it's time to update wsa with magisk support install

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#173] [Q&A] [answered]
This wsa with magisk and mindthegapps support to enable use of play store is excellent. 

When do you know it's time to fetch/pull or reclone the repo and build/run/install that wsa with magisk and mindthegapps support installation again?  

Is it created in such a way that once installed windows update and/or the microsoft store will maintain the winkey | "installed apps" | "windows subsystem for android" component?

**Comment by Howard20181** [comment, marked as answer]
There is no way to check for updates or automatic updates, it has to be done manually.

**Reply by myusrn** [reply]
Thanks for response. Any recommendations on how one goes about determining that there is a new and improved wsa with Magisk and mindthegapps update that warrants redoing the fetch+pull or fresh clone and run [ build/install ] process ?

**Reply by Howard20181** [reply]
There is no good way to check for updates to this repository's scripts
For Magisk you can subscribe to Releases updates
![image](https://user-images.githubusercontent.com/40033067/194733627-30af1074-521e-414b-a841-9ee22454d16e.png)

For MindThegapps, you don't need to keep the built-in one up-to-date at all times, the Play Store will update it
For WSA, subscribing to the Windows insider blog [RSS](https://blogs.windows.com/windows-insider/feed/) is a good option

**Reply by Howard20181** [reply]
https://blogs.windows.com/windows-insider/tag/windows-subsystem-for-android/
